aroquev/src/main/webapp/script.js

```diff
@@ -40,3 +40,36 @@ function calculateArmandoAge() {
     ageContainer.innerText = Math.abs(ageDt.getUTCFullYear() - 1970);
     
 }
+
+// Scripts for the image gallery
+var slideIndex = 1;
```

Comment:
Optional. A fun challenge would be to encapsulate all of this "slideshow" behavior into a module that exposes the methods to update the slides.

https://addyosmani.com/resources/essentialjsdesignpatterns/book/#modulepatternjavascript

If you decide to take this on, make sure it's in a followup PR, not this one.

Reply:
Sure, will do!